### DISCUSSION TITLE
Getting CREATE_FAILED | AWS::DataBrew::Dataset error when trying to create a Glue databrew dataset using AWS CDK

### DISCUSSION THREAD

**Discussion by kkshyp** [aws/aws-cdk#23468] [General]
Hi,

I am getting Getting CREATE_FAILED | AWS::DataBrew::Dataset error when trying to create a Glue databrew dataset.
I am trying to create a Glue databrew dataset using AWS CDK .

Resource handler returned message: "User: arn:aws:sts::XXXXXXXXX:assumed-role/cdk-XXXXXXXX-cfn-exec-role-XXXXXXXXX-us-east-1/AWSCloudFormation is not authorized to perform: glue:GetTable on resource: arn:aws:glue:us-east-1:tablename:catalog

I am creating Databrew datasets,recipe, project , job etc

I am trying to create a dataCatalogInputDefinition type data sets in aws cdk stack as below.
dataCatalogInputDefinition: {
catalogId: 'catalogId',
databaseName: 'databaseName',
tableName: 'tableName',
tempDirectory: {
bucket: 'bucket',

    // the properties below are optional
    key: 'key',
  },
},

**Comment by indrora** [comment]
This is an IAM permissions boundary problem. Double check if your organization has any enforced IAM permissions or policies that would prevent the assumed role for CDK/CFN from creating the resource itself.
